src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -296,8 +296,17 @@ public string PropertyName
         public abstract string TemplateHint { get; }
 
         /// <summary>
+        /// Gets a value that indicates whether this model should be validated.
+        /// </summary>
+        /// <value>Defaults to <c>true</c>.</value>
+        public abstract bool Validate { get; }
```

Comment:
👍 Good catch and it applies equally to the new `ShouldValidate` property addition. I'll file a bug on the API Checker that it misses this. I'll file the same bug on my brain but won't track that in GitHub.